src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -20,13 +20,45 @@ interface InputSpec {
   role?: 'axis' | 'stratification';
 }
 
+interface SectionSpec {
+  order: number;
+  title: string | ReactNode;
```

Comment:
```suggestion
  title: ReactNode;
```

(Strings are `ReactNode`s.)